docs/SETTINGS.rst

```diff
@@ -14,6 +14,7 @@ Bootstrap configuration
     -  **loop\_wait**: the number of seconds the loop will sleep. Default value: 10
     -  **ttl**: the TTL to acquire the leader lock. Think of it as the length of time before initiation of the automatic failover process. Default value: 30
     -  **maximum\_lag\_on\_failover**: the maximum bytes a follower may lag to be able to participate in leader election.
+    -  **synchronous\_mode**: turns on synchronous replication mode. Synchronous mode trades service availability for better durability guarantees. See `replication modes documentation <https://github.com/zalando/patroni/blob/master/docs/replication_modes.rst>`__ for details.
```

Comment:
I think the availability vs durability tradeoff should be explained in more details, since the whole point of Patroni is providing PostgreSQL HA. What about replacing the sentence starting with "Synchronous mode trades..." with "A replica running in synchronous mode is always in sync with the master. If the synchronous replica is not available, the master does not accept "write" transactions. Therefore, synchronous mode is a tradeoff between write availability and a strict data consistency between the master and the synchronous replica."